$NetBSD: patch-al,v 1.6 2006/06/26 11:39:01 rillig Exp $

Self-written configure tests are generally not of high quality. This one
fails to work on NetBSD 3.0.

--- configure.orig	2006-06-26 13:34:22.000000000 +0200
+++ configure	2006-06-26 13:36:17.000000000 +0200
@@ -3166,7 +3166,7 @@ fi
 done
 test -n "$YACC" || YACC="yacc"
 
-if ! which `echo $YACC | cut -f 1 -d ' '` >/dev/null 2>&1; then
+if false && ! which `echo $YACC | cut -f 1 -d ' '` >/dev/null 2>&1; then
    { { echo "$as_me:$LINENO: error: 'yacc' is missing on your system.
 You can get \`bison' from any GNU archive site." >&5
 echo "$as_me: error: 'yacc' is missing on your system.
